src/restore.c

```diff
@@ -2033,36 +2108,28 @@ get_dbOid_exclude_list(pgBackup *backup, parray *datname_list,
 
 /* Check that instance is suitable for incremental restore
  * Depending on type of incremental restore requirements are differs.
+ *
+ * Possible results:
+ *  (-4) POSTMASTER_IS_RUNNING
+ *  (-3) SYSTEM_ID_MISMATCH
+ *  (-2) BACKUP_LABEL_EXISTS
+ *  (-1) DEST_IS_NOT_OK
+ *   (0) DEST_OK
```

Comment:
I'd suggest not to mention numeric values here, as we can later add some more named variables and they will go out of sync with the comment.

Reply:
Ok